x/valaddr/client/cli/tx.go

```diff
@@ -0,0 +1,67 @@
+package cli
+
+import (
+	"github.com/celestiaorg/celestia-app/v6/x/valaddr/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+)
+
+// GetTxCmd returns the transaction commands for the valaddr module
+func GetTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Valaddr transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(
+		CmdSetFibreProviderIP(),
+	)
+
+	return cmd
+}
+
+// CmdSetFibreProviderIP broadcasts a MsgSetFibreProviderInfo transaction
+func CmdSetFibreProviderIP() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set-provider-ip [ip-address]",
```

Comment:
[nit] currently `MsgSetFibreProviderInfo` only contains the IP address but I think it may future-proof if this command was called


```suggestion
// CmdSetFibreProviderInfo broadcasts a MsgSetFibreProviderInfo transaction
func CmdSetFibreProviderInfo() *cobra.Command {
	cmd := &cobra.Command{
		Use:   "set-provider-info [ip-address]",
```

that way we can add more fields to ProviderInfo at a later date and not break the name of this command.

Note this suggestion doesn't fix all the instances in this file